### DISCUSSION TITLE
如何设置动态主题颜色切换

### DISCUSSION THREAD

**Discussion by strongyc** [DouyinFE/semi-design#921] [Q&A] [answered]
线上运行时，如何设置动态主题颜色切换

**Comment by pointhalo** [comment, marked as answer]
在 Semi DSM 中发布的 npm 主题包，都带有已构建好的 css 产物。如果你需要动态切换的话，可以自行动态加载不同 npm 包的产物。
例如@semi-bot/semi-theme-nyx-c 这个包   node_modules/@semi-bot/semi-theme-nyx-c/semi.min.css 这个就是 构建好的css包。你可以直接读取 对应npm 的 unpkg cdn，直接动态创建 link 标签加载，
